src/views/Home.vue

```diff
@@ -1,20 +1,87 @@
 <template>
-  <div class="home">
-    <h1>欢迎来到 Enspire!</h1>
-    <img
+  <div class="home" id="mainpage">
+    <div class="art-welcome">
+      <div class="art-title">
+        <h1 class="art-titleTx1">Welcome to</h1>
+        <h1 class="art-titleTx2">Enspire</h1>
+      </div>
+      <v-carousel class="art-banner">
+        <v-carousel-item
+          v-for="(banner, i) in banners"
+          :key="i"
+          :src="banners.src"
+        ></v-carousel-item>
+      </v-carousel>
+      <!--img
       alt="Enspire logo"
       class="logo"
       src="@/assets/img/Enspire-logo.svg"
       width="100px"
-    />
+    /-->
+    </div>
   </div>
 </template>
 
 <style scoped>
-.home {
+@font-face {
+  font-family: title;
+  src: url("../assets/fonts/Britannic_Bold_T.ttf");
```

Comment:
I don't think this font fits well with the site—maybe use a sans-serif font for a more professional look?